test/map.spec.js

```diff
@@ -381,8 +385,10 @@ test('getMapTraversabilityInCells: returns correctly with CPTL=1', t => {
   const blank = tileTypes.REGULAR_FLOOR;
 
   // initialize current player as blue
```

Comment:
@billmwong I'm having trouble with this test. Could you please help me with your rewiring mastery?